README.md

```diff
@@ -1 +1,15 @@
-# clojure-tic-tac-toe
\ No newline at end of file
+# Unbeatable Tic-Tac-Toe in Clojure
```

Comment:
Can you add some more details in your readme. How do we run tests? What are some dependencies we need if any?